src/couch/src/couch_os_process.erl

```diff
@@ -157,6 +157,7 @@ init([Command, Options, PortOptions]) ->
     couch_io_logger:start(os:getenv("COUCHDB_IO_LOG_DIR")),
     PrivDir = couch_util:priv_dir(),
     Spawnkiller = "\"" ++ filename:join(PrivDir, "couchspawnkillable") ++ "\"",
+    couch_log:info("OS Process Start (command):: ~p", [Command]),
```

Comment:
This might be a stray / accidental log line ;-)

Though not a bad idea but in a different PR

Reply:
Oh yes, done a `git commit -a ...` without thinking...

Reply:
I've done that many times as well :-)